source/material_model/rheology/strain_dependent.cc

```diff
@@ -147,6 +147,27 @@ namespace aspect
                            "for background material and compositional fields, "
                            "for a total of N+1 values, where N is the number of compositional fields. "
                            "If only one value is given, then all use the same value.  Units: None.");
+
+        prm.declare_entry ("Strain healing mechanism", "no healing",
+                           Patterns::Selection("no healing|temperature dependent"),
+                           "Whether to apply strain healing to plastic yielding and viscosity terms, "
+                           "and if yes, which method to use. The following methods are available:"
+                           "\n\n"
+                           "\\item ``no healing'': No strain healing is applied. "
+                           "\n\n"
+                           "\\item ``temperature dependent'': Purely temperature dependent "
+                           "strain healing applied to plastic yielding and viscosity terms, similar "
+                           "to the temperature-dependent Frank Kamenetskii formulation, computes "
+                           "strain healing as removing strain as a function of temperature and time"
+                           "as done in Fuchs and Becker, 2019, for mantle convection");
+
+        prm.declare_entry ("Strain healing recovery timescale", "1.e-15", Patterns::Double(0),
```

Comment:
```suggestion
        prm.declare_entry ("Strain healing temperature dependent recovery timescale", "1.e-15", Patterns::Double(0),
```

Reply:
Adding in `temperature dependent` to the above parameters to avoid confusion when additional strain healing mechanisms are added.

Reply:
Also, see my above suggestion for calling this variable a `recovery rate`.